libraries/rush-lib/src/cli/actions/LinkPackageAction.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import type { RushCommandLineParser } from '../RushCommandLineParser';
+import type { RushConfigurationProject } from '../../api/RushConfigurationProject';
+import { BaseSymlinkPackageAction } from './BaseSymlinkPackageAction';
+import type { RushConnect } from '../../utilities/RushConnect';
+
+export class LinkPackageAction extends BaseSymlinkPackageAction {
+  public constructor(parser: RushCommandLineParser) {
+    super({
+      actionName: 'link-package',
+      summary:
+        '(EXPERIMENTAL) Simulate installation of a locally built project, affecting specific projects.',
+      documentation:
+        'This command enables you to test a locally built project by creating a symlink under a consuming' +
+        ' project\'s node_modules folder to simulate installation.  The implementation is similar to "pnpm link"' +
```

Comment:
' by simulating its installation inside projects'

```suggestion
        'This command enables you to test a package that exists in a folder outside of this repo' +
        ' by creating a symlink under a consuming project\'s node_modules folder to simulate installation.  The implementation is similar to "pnpm link"' +
```